assets/js/services/auth_service.js

```diff
@@ -1,4 +1,5 @@
 app.factory("AuthService", ["$http", "$location", function($http, $location) {
+
```

Comment:
these spaces seem inconsistent with what we've got in the controllers and elsewhere.